site/_posts/2020-09-03-oidc-auth-api-usage.md

```diff
@@ -78,15 +83,43 @@ oidc_client_secret="12345678-1234-1234-1234-01234567abcd"
 # OIDCClientID from the OpenID-Connect configuration.
 oidc_client_id="my-oidc-client"
 
-# OIDCOauthIntrospectionEndpoint from the OpenID-Connect configuration
-oidc_auth_introspection_endpoint="https://keycloak.example.com/auth/realms/miq/protocol/openid-connect/token/introspect"
-
-# OIDCProviderTokenEndpoint from the OpenID-Connect configurationt
-# Sometimes OIDCOauthIntrospectionEndpoint without the trailing "introspect"
-oidc_provider_token_endpoint="https://keycloak.example.com/auth/realms/miq/protocol/openid-connect/token/"
 ```
 
-## Example: Using a JWT Token
+Each OpenID-Connect Provider defines its own endpoints for token generation and introspection. Consequently
+the value of these variables depend on the OpenID-Connect Provider being used.
+
+**Example for IBM's Identiy Access Management (IAM):**
```

Comment:
typo: `Identiy`

Reply:
@chessbyte Good catch. It's odd the spell checker doesn't flag this.